var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -68,6 +65,12 @@ class Paraview(CMakePackage, CudaPackage):
     # See commit: https://gitlab.kitware.com/paraview/paraview/-/commit/798d328c
     conflicts('~opengl2', when='@5.5:')
 
+    with when('+cuda'):
```

Comment:
@alalazo I wonder if there is any way to further concise this

Reply:
You could use a loop inside the `with`:
```python
for _arch in range(10, 14):
    conflicts('cuda_arch=%d' % _arch, msg='ParaView requires cuda_arch >= 20')
 ```

Reply:
@sethrj answer is correct. I would be fine with both solutions.